src/content/1.7/modules/creation/good-practices.md

```diff
@@ -29,33 +29,32 @@ menuTitle: Good practices
 - Prefix what belongs to you:
   - modules
   - configuration parameters (ps_configuration)
+  - Smarty variables
   - tables
   - CSS classes
   
-- Consider deleting your configurations, tables, and other entities unique to your product during the de-installation of your modules.
+- Consider deleting your configurations, tables, admin tabs, and all other entities unique to your product during the uninstallation of your modules.
 
 - You shouldn't use HTML code in your PHP code. Here are alternatives to separate the view into your code:
   - display content with [Smarty][display-content-front-office]
   - use [helpers][adding-configuration-page] for your configuration pages
   - include [PrestUI](https://github.com/Scritik/prestui), a graphic library created by the community for your configuration pages.
 
-- Your module interface has to match the back office interface. Customized interfaces aren’t validated/accepted. Please use the alternatives explained before to help you create an interface that matches merchants' BO.
+- We recommend to create your module's interface so it matches [PrestaShop's UI kit](http://build.prestashop.com/prestashop-ui-kit/).
```

Comment:
👍

Reply:
I was told throught slack to not communicate that link.
So which one ?